data-prepper-plugins/aws-plugin/src/main/java/org/opensearch/dataprepper/plugins/aws/AwsSecretPlugin.java

```diff
@@ -21,7 +21,8 @@
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.TimeUnit;
 
-@DataPrepperExtensionPlugin(modelType = AwsSecretPluginConfig.class, rootKeyJsonPath = "/aws/secrets",
+@DataPrepperExtensionPlugin(modelType = AwsSecretPluginConfig.class, deprecatedRootKeyJsonPath = "/aws/secrets_manager",
+        rootKeyJsonPath = "/aws/secrets",
```

Comment:
Should "/aws/secrets" be the deprecatedRootKeyJsonPath instead?

Reply:
Thanks for the catch!